test/core/test/nested-test.test.ts

```diff
@@ -3,28 +3,28 @@ import { describe, expect, test } from 'vitest'
 test('nested test should throw error', () => {
   expect(() => {
     test('test inside test', () => {})
-  }).toThrowErrorMatchingInlineSnapshot(`[Error: Nested tests are not allowed]`)
+  }).toThrowErrorMatchingInlineSnapshot(`[Error: Calling the "test" or "it" function inside another "test" or "it" function is not allowed. Please, put it inside "describe" so it can be properly collected.]`)
 
   expect(() => {
     test.each([1, 2, 3])('test.each inside test %d', () => {})
-  }).toThrowErrorMatchingInlineSnapshot(`[Error: Nested tests are not allowed]`)
+  }).toThrowErrorMatchingInlineSnapshot(`[Error: Calling the "test" or "it" function inside another "test" or "it" function is not allowed. Please, put it inside "describe" so it can be properly collected.]`)
```

Comment:
```suggestion
  }).toThrowErrorMatchingInlineSnapshot(`[Error: Calling the "test" or "it" function inside another "test" or "it" function is not allowed. Please put it inside "describe" so it can be properly collected.]`)
```